packages/preview/gantty/0.4.0/README.md

```diff
@@ -0,0 +1,85 @@
+# Gantty
+
+Gantty is a typst library for creating Gantt charts using datetimes.
+
+View the manual [here](https://john_t.gitlab.io/typst-gantty/manual.pdf).
+
+## Features
+
+- Works with real dates.
+- Customizable styling.
+- Extensible to support multiple languages.
+
+## Why not typst-timeliney
+
+[Typst-timeliney](https://typst.app/universe/package/timeliney) is another
+excellent typst Gantt chart creator. The main differences between this library
+and typst-timeliney is that Gantty uses datetimes to work with real dates,
+whereas typst-timeliney works with coordinates. Look at the typst-timeliney
+documentation and decide which is more fitting for your use.
+
+## Example
+
+![A Gantt chart](https://gitlab.com/john_t/typst-gantty/-/raw/master/example/gantt.svg)
+
+index.typ:
+
+```typst
+#import "@preview/gantty:0.3.0": gantt
```

Comment:
```suggestion
#import "@preview/gantty:0.4.0": gantt
```

Reply:
(This is why I like Typst where I can simply import the version number from a TOML file!)

Thank you, though, for noticing this, `:)`.